pkgs/by-name/om/omekasy/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "omekasy";
+  version = "1.2.3";
+
+  src = fetchFromGitHub {
+    owner = "ikanago";
+    repo = "omekasy";
+    rev = "v${version}";
+    hash = "sha256-oPhO+gRWrwgABc+gGXnIC519F5XVvewUHo2y54RoE4U=";
+  };
+
+  cargoSha256 = "sha256-6GjNn7FAcAihqNhPD18sUFe40ZQwXmFEQmoZNZL2trQ=";
+
+  meta = {
+    description = "𝘾𝙤𝙢𝙢𝙖𝙣𝙙 𝙡𝙞𝙣𝙚 𝙖𝙥𝙥𝙡𝙞𝙘𝙖𝙩𝙞𝙤𝙣 𝕥𝕙𝕒𝕥 𝕔𝕠𝕟𝕧𝕖𝕣𝕥𝕤 𝕒𝕝𝕡𝕙𝕒𝕟𝕦𝕞𝕖𝕣𝕚𝕔 𝕔𝕙𝕒𝕣𝕒𝕔𝕥𝕖𝕣𝕤 𝒕𝒐 𝒗𝒂𝒓𝒊𝒐𝒖𝒔 𝒔𝒕𝒚𝒍𝒆𝒔 𝚍𝚎𝚏𝚒𝚗𝚎𝚍 𝚒𝚗 𝚄𝚗𝚒𝚌𝚘𝚍𝚎. ";
```

Comment:
Let’s not play with this here? Also no trailing dot and space

Reply:
Mmh. I suppose it might break search? I'm not sure the "various styles" is quite descriptive enough, with this playful description it's obvious what the package does.

What do you think of providing a few examples in longDescription? It [wouldn't be the first](https://github.com/NixOS/nixpkgs/blob/9b1b4ecaf3bbebd6679cfceaa267d6b56e3f1e0e/pkgs/by-name/li/little_boxes/package.nix#L48-L50) to use weird characters there.

Anyway, fixed the `.  `. Ty. Wouldn't mind this getting merged as is.

Reply:
(In case this wasn't obvious from the comment, I did change the description to be all ascii.)